release/ray_release/byod/build_ray.py

```diff
@@ -19,8 +19,8 @@ def build_ray(tests: List[Test]) -> None:
     """
     Builds ray and ray-ml images for PR builds
     """
-    if not _is_pr():
-        logger.info("Not a PR, skipping build_ray")
+    if not _is_pr() or os.environ.get("RAY_CI_V2"):
```

Comment:
how is this being set today?

maybe change it to `RAYCI_V2`? We commonly use `RAYCI_` for pipeline wide custom env vars.

Reply:
It's set in the pipeline steps definition https://buildkite.com/ray-project/release/settings/steps, e.g. RAYCI_V2=1 python ray_release/scripts/build_pipeline.py